cartoframes/context.py

```diff
@@ -182,6 +182,7 @@ def __init__(self, base_url=None, api_key=None, creds=None, session=None,
                 session=session
             )
         self.sql_client = SQLClient(self.auth_client)
+        self.copyClient = CopySQLClient(self.auth_client)
```

Comment:
let's stick to `lower_case_with_underscores` for variables

Reply:
:+1: